### ISSUE TITLE
Frame-based training

### ISSUE THREAD

**Issue by Jay-IPL** [wjn922/ReferFormer#36]
Hi authors~ I noticed in your paper you mentioned you put the temporal dimension into the batch dimension and extract visual features independently. Thus, I think the whole model is frame-based training except when using video-swin as the backbone. Is my understanding right?

**Comment by iseunghoon** [comment]
> Hi authors~ I noticed in your paper you mentioned you put the temporal dimension into the batch dimension and extract visual features independently. Thus, I think the whole model is frame-based training except when using video-swin as the backbone. Is my understanding right?

I agree with your point. Aside from Video Swin, there is no fusion along the temporal axis, right?"
